03_Cloud_Functions/prueba_alertas.py

```diff
@@ -35,11 +36,26 @@ def pubsub_to_bigquery(event, context):
     # BigQuery
     try:
         bq_client = bigquery.Client(project=project_id)
-        errors = bq_client.insert_rows_json(table, [message])
-        if errors == []:
-            logging.info("New rows have been added into BigQuery table.")
-        else:
-            logging.error("Encountered errors while inserting rows into BigQuery table.: {}".format (errors))
+        try:
+            bq_client.get_table(table_id)
+            print("Table {} already exists.".format(table_id))
+        except NotFound:
+            schema = [
+            bigquery.SchemaField("device_id", "STRING", mode="REQUIRED"),
+            bigquery.SchemaField("device_name", "STRING", mode="REQUIRED"),
+            bigquery.SchemaField("timestamp", "STRING", mode="REQUIRED"),
+            bigquery.SchemaField("kw", "STRING", mode="REQUIRED"),
+            bigquery.SchemaField("processingTime", "STRING", mode="REQUIRED"),
```

Comment:
Si no recuerdo mal, ayer en el dataflow, cambiamos este atributo. 

```suggestion
            bigquery.SchemaField("processing_time", "STRING", mode="REQUIRED"),
```